templates/rack_mini_profiler.rb

```diff
@@ -0,0 +1,5 @@
+if ENV["RACK_MINI_PROFILER"].to_i > 0
```

Comment:
@jferris has that change landed yet?

Is this PR good to go without it, or does it depend on the resolution of that PR?